### PR TITLE
Changelog for 10.23.3 

### PR DESCRIPTION
**Pull request by PeterYinusa**
## Explanation

Changelog for 10.23.3

## Pre-merge author checklist

- [ ] I've clearly explained:
  - [ ] What problem this PR is solving
  - [ ] How this problem was solved
  - [ ] How reviewers can test my changes
- [ ] Sufficient automated test coverage has been added

## Pre-merge reviewer checklist

- [ ] Manual testing (e.g. pull and build branch, run in browser, test code being changed)
- [ ] PR is linked to the appropriate GitHub issue
- [ ] **IF** this PR fixes a bug in the release milestone, add this PR to the release milestone

If further QA is required (e.g. new feature, complex testing steps, large refactor), add the `Extension QA Board` label.

In this case, a QA Engineer approval will be be required.

### CODE REVIEW

**Review by brad-decker — DISMISSED** [review]

**Comment by Gudahtt** [comment]
I suspect this should be targeting `develop` rather than `master`. #17122 suggests this will land on `develop`, and in general any commit to `master` is treated as a release so we should avoid doing that for non-releases.

**Comment by PeterYinusa** [comment]
> I suspect this should be targeting `develop` rather than `master`. #17122 suggests this will land on `develop`, and in general any commit to `master` is treated as a release so we should avoid doing that for non-releases.

Yes, I'll get the base updated once master sync has merged.
Initially this branch targeted 10.23.3, but the base automatically changed to master as the RC was merged ahead of this change

**Review by danjm — DISMISSED** [review]

File: CHANGELOG.md [hunk 1]
```diff
@@ -7,6 +7,8 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [Unreleased]
 
 ## [10.23.3]
+### Changed
```
Comment (Gudahtt) [inline CHANGELOG.md:10]: Nit: there is a `Removed` category for removals. We don't use it often, but it seems appropriate here

```suggestion
### Removed
```

**Review by Gudahtt — DISMISSED** [review]
LGTM!

**Review by Gudahtt — APPROVED** [review]

**Review by danjm — APPROVED** [review]

**Pull request merged**